Dynamic_Bar.lua

```diff
@@ -100,28 +92,20 @@ function Type:Setup(icon)
  		values.triggerFunc = fnc
  	end
 
- 	icon:SetInfo("state;", 1)
 	icon:Update()
 end
 
 TMW:RegisterCallback("TMW_CONFIG_ICON_TYPE_CHANGED", function(event, icon, type, oldType)
 	local icspv = icon:GetSettingsPerView()
 
-	if type == Type.type then
```

Comment:
I tweaked this handler to avoid it potentially stomping on the same version of it that exists in the `value` icon type. This version is a bit toned down - it will only setup the default texts on the icon when going from an unconfigured icon to your icon type, instead of from ANY icon type. It will also no longer attempt to revert these when switching away from your icon type.